src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,26 @@
+import { space, SpacingUnit } from "@artsy/palette"
+import styled from "styled-components"
+import { media } from "../../Components/Helpers"
+
+interface PaddingProps {
+  sizeXS?: SpacingUnit | 0
+  size?: SpacingUnit | 0
+}
```

Comment:
If `size` and `sizeXS` refer explicitly to horizontal left and right padding and additional api methods outside of styled-system are necessary, then it would be clearer to follow `styled-system`'s naming model as closely as possible: 

```js
pxXS: SpacingUnit
px: SpacingUnit